src/components/ChatApp.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Header from './Header';
+import ChatMessages from './ChatMessages';
+import ChatInput from './ChatInput';
+import chatData from '../assets/data/ChatData.json'
+import userData from '../assets/data/UserData.json'; 
+
+const ChatContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 800px;
+  max-width: 400px;
+  margin: 0 auto;
+  border: 1px solid #ddd;
+  border-radius: 10px;
+  overflow: hidden;
+`;
+
+const ChatApp = () => {
+  const [messages, setMessages] = useState([]);
+  const [users, setUsers] = useState({});
```

Comment:
```suggestion
  const [messages, setMessages] = useState(chatData);
  const [users, setUsers] = useState(userData);
```
`useEffect`를 사용하지 않고 이렇게 초기값으로 넣어 줄 수 있습니다!